lib/sensors/INA_226.cpp

```diff
@@ -0,0 +1,79 @@
+#include "mbed.h"
+#include "INA_226.h"
+
+INA_226::INA_226(int i2c_frequency, PinName SDA_pinname, PinName SCK_pinname, u_int8_t sensor_address, float max_expected_current, float shunt_resistance)
+{
+    shunt_resistance_ = shunt_resistance;
+    max_expected_current_ = max_expected_current;
+    sensor_address_ = sensor_address << 1;
+
+    i2c_(SDA_pinname, SCK_pinname);
+    i2c_.frequency(i2c_frequency);
+}
+
+INA_226::~INA_226(){}
+
+//TODO: may need to adjust function to take sensor slave addresses as there many be many current sensors
+float Safety_Board::getCurrentData()
+{
+    //may need char16_t?
+    u_int8_t cmd[3] = {CURRENT_REGISTER};

```

Comment:
cmd does not need to have 3 element, 2 is enough since the read is going to overwrite the value that was set to write